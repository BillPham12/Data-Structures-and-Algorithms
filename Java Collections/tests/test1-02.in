1312
213123
d123d12
312d312d2
d3123d12
d312d312d3
12d312d312
d312d12
