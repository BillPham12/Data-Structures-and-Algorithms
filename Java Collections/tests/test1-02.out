d312d12
12d312d312
d312d312d3
d3123d12
312d312d2
d123d12
213123
1312
